mycli/myclirc

```diff
@@ -77,6 +77,9 @@ login_path_as_host = False
 # and using normal tabular format otherwise. (This applies to statements terminated by ; or \G.)
 auto_vertical_output = False
 
+# keyword casing preference. Possible values "lower", "upper", "auto"
+keyword_casing = upper
```

Comment:
I was thinking more along the lines of preserving current mycli behavior in defaults, but it's true, auto sounds like a better default. I'll change that.